src/hotspot/os/linux/os_linux.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1999, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright change seems to be the only change in this file, and should be reverted.